client/blocks/inline-help/inline-help-search-results.jsx

```diff
@@ -123,13 +130,26 @@ function HelpSearchResults( {
 	};
 
 	const renderHelpLink = ( result ) => {
-		const { link, key, title, support_type = SUPPORT_TYPE_API_HELP, icon = 'domains' } = result;
+		const {
+			link,
+			key,
+			title,
+			support_type = SUPPORT_TYPE_API_HELP,
+			icon = 'domains',
+			post_id,
+		} = result;
 		const resultIndex = searchResults.findIndex( ( r ) => r.link === link );
 
 		const classes = classNames( 'inline-help__results-item', {
 			'is-selected': selectedResultIndex === resultIndex,
 		} );
 
+		// Unless searching with Inline Help or on the Purchases section, hide the
+		// "Managing Purchases" documentation link for users who have not made a purchase.
+		if ( post_id === 111349 && ! isSearching && ! hasPurchases && sectionName !== 'purchases' ) {
```

Comment:
`! isSearching` only applies while the search is happening (not once the results are shown). If a user without a purchase searches "purchases", "refunds", etc… then this document will be hidden. `hasAPIResults` should fix that.
```suggestion
		if ( post_id === 111349 && ! isSearching && ! hasAPIResults && ! hasPurchases && sectionName !== 'purchases' ) {
```

Reply:
Thank you - that makes sense! :)